pkgs/markdown/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 7.3.1-wip
+
+* Update the README link to the markdown playground
+  (https://dart-lang.github.io/tools).
```

Comment:
We should get a redirect in the old place, too.

https://github.com/dart-archive/dart-lang.github.io is how you do it. Not sure why it was archived.

Reply:
You probably mean https://github.com/dart-lang/markdown (the archived repo above was for package:io).

+1 to a re-direct.

Reply:
I DON'T! You can put anything in `dart-lang.github.io/XYZ`. Even `/markdown` if the repo isn't there.